src/wp-admin/includes/class-plugin-installer-skin.php

```diff
@@ -259,8 +259,9 @@ private function do_overwrite() {
 		$blocked_message  = '<p>' . esc_html__( 'The plugin cannot be updated due to the following:' ) . '</p>';
 		$blocked_message .= '<ul class="ul-disc">';
 
-		$requires_php = isset( $new_plugin_data['RequiresPHP'] ) ? $new_plugin_data['RequiresPHP'] : null;
-		$requires_wp  = isset( $new_plugin_data['RequiresWP'] ) ? $new_plugin_data['RequiresWP'] : null;
+		$requires_php     = isset( $new_plugin_data['RequiresPHP'] ) ? $new_plugin_data['RequiresPHP'] : null;
+		$requires_wp      = isset( $new_plugin_data['RequiresWP'] ) ? $new_plugin_data['RequiresWP'] : null;
+		$requires_network = isset( $new_plugin_data['Requires Network'] ) ? $new_plugin_data['Requires Network'] : null;
```

Comment:
```suggestion
		$requires_network = isset( $new_plugin_data['RequiresNetwork'] ) ? $new_plugin_data['RequiresNetwork'] : null;
```

Should be `RequiresNetwork`? What do you think @petitphp

Reply:
@mukeshpanchal27 Yes, you're right, I've mixed up the naming each time. Thanks for catching that !